homeassistant/components/habitica/sensor.py

```diff
@@ -306,4 +306,30 @@ async def async_added_to_hass(self) -> None:
                     DOMAIN,
                     f"deprecated_task_entity_{self.entity_description.key}",
                 )
+
+            if (
+                self.enabled
```

Comment:
I'm having problem understanding this one.

How are attributes related to how this sensor is enabled or not?

I think you are mixing up state attributes and entity state?

Reply:
I think the idea is to not show deprecation warnings for sensors that are disabled but it may be unnecessary since there is already a check for `entity_used_in(self.hass, entity_id)`

Reply:
These sensors purpose is to use the dictionary data that is dumped into the state attributes, for example for templating as [described in the docs](https://www.home-assistant.io/integrations/habitica/#templating). So I can safely assume the user is accessing the sensors attributes if it is used in automations and scripts. Otherwise I don't need to bother them with an issue. 

I think it is not possible to check for use in templates, so should I just in case, also send a deprecation warning to the logger?

Reply:
Another possibility I haven't thought of yet, is to completely remove those sensors as the sole count of habits and rewards is not very useful